### ISSUE TITLE
Add an 'Accessibility' icon that can be used in Discussions

### ISSUE THREAD

**Issue by ghostinhershell** [github/gemoji#258]
## Request
I'd like to have a new :accessibility: icon/emoji added to Discussions, similar to what's currently available in Slack. Octicons does have an accessibility-inset icon that would work: https://primer.style/octicons/accessibility-inset-16. Can anyone point me in the right direction so that this can be added and available for use in Discussions?

## Use Case: 
In an effort to help clarify what the Accessibility category in the Community on Discussions is (and reduce the number of mis-categorized posts in that area of the community), I was thinking that updating the icon/emoji for that category made sense. Right now, the Accessibility community category's emoji is a 👍 which is non-descriptive and isn't useful.

I initially proposed changing it to something like ♿ since that's the only Accessibility-related icon available in Discussions at the moment. However, @benjiallen brought up the valid point that it's not the most inclusive icon to use. 

## Icon
![accessibility-inset](https://user-images.githubusercontent.com/14021469/214177248-2d72623b-322f-429b-9726-6069e1dfd718.svg)

**Comment by mislav** [comment]
Hi, the aim of gemoji library is to _only_ cover Unicode codepoints and not maintain any “custom” emoji anymore https://github.com/github/gemoji/pull/158

I do agree that ♿ is not inclusive of different kinds of accessibility. If your goal is to add an additional (non-Unicode) emoji to GitHub.com so it can be used in Discussions, I suggest that this gets done in GitHub's own codebase, rather than in this library.
